### DISCUSSION TITLE
article/jqtuqouj/

### DISCUSSION THREAD

**Discussion by giscus** [ermaozi/ermao.net#4] [Announcements]
# article/jqtuqouj/

之前我的博客一直使用 wordpress，但是由于 wordpress 太过笨重、维护难度大、国内访问速度慢、还需要花钱租用后端服务器。所以我决定迁移到 github pages 重搞一个博客。 但是之前放在公网上的文章链接都是 wordpress 的，我又舍不得放弃这部分流量，所以用 cloudflare 进行URL重定向，原来的访问直接 301 跳...

https://www.ermao.net/article/jqtuqouj/

**Comment by mhcms** [comment]
## 1
### 2
